src/utils/tenant.ts

```diff
@@ -0,0 +1,79 @@
+import { runMigrationsOnTenant } from './migrate'
+import { pool } from './multitenant-db'
+
+interface TenantConfig {
+  anonKey: string
+  databaseUrl: string
+  jwtSecret: string
+  serviceKey: string
+}
+
+const tenantConfigCache: {
+  [tenantId: string]: TenantConfig
```

Comment:
Make this into a Map instead of an object? Faster and more explicit.

Reply:
Done.